qiime/normalize_table.py

```diff
@@ -20,16 +24,18 @@
 def normalize_CSS(input_path, out_path, output_CSS_statistics):
     """performs metagenomeSeq's CSS normalization on a single raw abundance OTU matrix
     """
-    base_fname, ext = splitext(input_path)
-    json_infile = base_fname+'_json.biom'
-    open(str(json_infile),'w').write(load_table(input_path).to_json('forR'))
+    tmp_bt = load_table(input_path) 
 
     if output_CSS_statistics:
         base_fname, ext = splitext(out_path)
         output_CSS_statistics = base_fname+'_CSS_statistics.txt'
-    
-    run_CSS(json_infile, out_path, output_CSS_statistics=output_CSS_statistics)
-    remove(json_infile)
+
+    with tempfile.NamedTemporaryFile(dir=get_qiime_temp_dir(),
+                                 prefix='QIIME-differential-abundance-temp-table-',
```

Comment:
Please update the prefix here and in the rest of the code to be `QIIME-normalize-table-temp-table-`.